lms/lib/comment_client/comment.py

```diff
@@ -30,6 +30,11 @@ class Comment(models.Model):
     def thread(self):
         return Thread(id=self.thread_id, type='thread')
 
+    @property
+    def context(self):
```

Comment:
This is still useful to allow the signal handlers to work for both threads and comments. If it were removed, the `getattr(post, 'context', 'course')` call would always return `'course'` for comments.